storage/redis/client.go

```diff
@@ -117,6 +118,30 @@ func (c *redisClient) Delete(key storage.Key) error {
 }
 
 // Close closes a redis client
-func (c *redisClient) Close() error {
+func (c *Client) Close() error {
 	return c.db.Close()
 }
+
+// GetAll is the bulk method for gets from the redis data store
+func (c *Client) GetAll(keys storage.Keys) (storage.Values, error) {
+	ks := []string{}
+	for _, v := range keys {
+		ks = append(ks, v.String())
```

Comment:
As `len(keys)` is known, isn't it more effective from performance point of view, to make a fixed-size slice instead of appending all the values one by one?

Same for `values`.

Reply:
yes it would be more efficient. I'll make the change